example-get-started/generate.sh

```diff
@@ -301,16 +301,26 @@ EOF
   if [ $OPT_DVC_TRACKED_METRICS == "true" ]; then
     dvc stage add -n evaluate \
       -d src/evaluate.py -d model.pkl -d data/features \
-      -m eval/live/metrics.json -o eval/live/plots \
+      -o eval/live/metrics.json -o eval/live/plots \
       python src/evaluate.py model.pkl data/features
   else
     dvc stage add -n evaluate \
       -d src/evaluate.py -d model.pkl -d data/features \
-      -M eval/live/metrics.json -O eval/live/plots \
```

Comment:
do we need to add `-O eval/live/metrics.json` here?

Reply:
need also to double check the semantics - `-o` vs `-O`

Reply:
No, and adding it would be inconsistent with the approach in https://github.com/iterative/dvclive/issues/456 and https://github.com/iterative/dvclive/pull/679. I don't have a strong opinion that we need to drop these outputs, but whatever convention we suggest, let's try to be consistent with it.

Reply:
kk, I'm fine with that. That was to double check primarily.